app/views/layouts/application.html.erb

```diff
@@ -72,13 +72,17 @@
       document.body.className = ((document.body.className) ? document.body.className + " js-enabled" : "js-enabled");
     </script>
 
-    <!-- Google Tag Manager (noscript) -->
-    <% if cookies[:accept_analytics_cookies] %>
+    <% if cookies[:accept_analytics_cookies] == "on" %>
+      <!-- Google Tag Manager (noscript) -->
       <noscript>
```

Comment:
For this to be relevant the user would have to be following the very specific behaviour of having JS on for the banner and then turning it off, or having it off the whole time and explicitly going to the cookies page to accept cookies. But just in case 🙂